wgengine/router/osrouter/router_linux.go

```diff
@@ -86,6 +87,7 @@ type linuxRouter struct {
 	cmd commandRunner
 	nfr linuxfw.NetfilterRunner
 
+	mu              sync.Mutex
```

Comment:
I think this may need to cover `snatSubnetRoutes`, `netfilterMode`, and `netfilterKind` too, from looking at the guts of `setNetfilterMode`. (Maybe others? But I didn't see any others)